src/Augurk/Controllers/V2/AugurkController.cs

```diff
@@ -137,7 +143,10 @@ public async Task<ActionResult> Import(IFormFile file)
                 System.IO.File.Delete(filePath);
 
                 // Set the expirations as configured
-                await _expirationManager.ApplyExpirationPolicyAsync(await _configurationManager.GetOrCreateConfigurationAsync());
+                //await _expirationManager.ApplyExpirationPolicyAsync(await _configurationManager.GetOrCreateConfigurationAsync());
```

Comment:
Misschien hier even een TODO bij zetten zodat we dat niet uit het oog verliezen?